dvc/tree/azure.py

```diff
@@ -20,6 +20,7 @@ class AzureTree(BaseTree):
     PATH_CLS = CloudURLInfo
     REQUIRES = {
         "azure-storage-blob": "azure.storage.blob",
+        "azure-identity": "azure.identity",
```

Comment:
what are the dependencies of this sub-package? we had some problems before with azure-core. We need to check that this one doesn't pull a lot of stuff with it.

Reply:
From https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/identity/azure-identity/setup.py
```
    install_requires=[
        "azure-core<2.0.0,>=1.0.0",
        "cryptography>=2.1.4",
        "msal<2.0.0,>=1.6.0",
        "msal-extensions~=0.3.0",
        "six>=1.6",
    ],
    extras_require={
        ":python_version<'3.0'": ["azure-nspkg"],
        ":python_version<'3.5'": ["typing"],
    }
```

Reply:
Yep, thanks! should be fine I think .... @efiop could you confirm?